tests/Integration/Behaviour/Features/Context/Domain/AttributeFeatureContext.php

```diff
@@ -149,7 +155,8 @@ private function createAttributeUsingCommand(
         int $attributeGroupId,
         array $localizedValues,
         string $color,
-        array $shopIds
+        array $shopIds,
+        ?string $filePath = null
```

Comment:
```suggestion
        string $filePath = null
```
I believe this is never null